src/methods/photos.js

```diff
@@ -87,11 +87,12 @@ export default function photos(): Object {
         query: options.query,
         w: options.width,
         h: options.height,
-        c: options.cacheBuster || new Date().getTime() // Avoid ajax response caching
+        c: options.cacheBuster || new Date().getTime(), // Avoid ajax response caching
+        count: options.count
       };
 
       Object.keys(query).forEach(key => {
-        if (!query[key]) {
+        if (!query[key] || query[key] == "") {
```

Comment:
Hm. Could you elaborate on this conditional, and why you added it?

I can see a potential application if you tie a search input in your app to the search query. Deleting all the characters would cause a search to be submitted with the query `''`, but it seems to me in this case you actually don't want to perform a search, since it would return a totally random photo.